src/js/thirdparty/detect-libc.linux.js

```diff
@@ -1,10 +1,13 @@
 // Hardcoded module "detect-libc" for linux
+
+const is_glibc = process.report.getReport().header.glibcVersionCompiler !== undefined;
```

Comment:
oh this is kind of silly tbh.  process.report.getReport() is kind of expensive, we could have a constant here.